packages/jest-cli/src/reporters/VerboseReporter.js

```diff
@@ -21,14 +22,10 @@ const DefaultReporter = require('./DefaultReporter');
 const chalk = require('chalk');
 const {ICONS} = require('../constants');
 
-type Options = {|
-  expand: boolean,
-|};
-
 class VerboseReporter extends DefaultReporter {
-  _options: Options;
+  _options: GlobalConfig;
 
-  constructor(options: Options) {
+  constructor(options: GlobalConfig) {
```

Comment:
I deliberately changed this in the past to *not* include all config options. But I guess it is fine – can we change this variable name to `_globalConfig` and `globalConfig`?